hw/ip/clkmgr/dv/cov/clkmgr_cov_unr_excl.el

```diff
@@ -0,0 +1,464 @@
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_io_div2_meas_ctrl_en_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_io_div2_meas_ctrl_shadowed_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_io_div4_meas_ctrl_en_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_io_div4_meas_ctrl_shadowed_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_main_meas_ctrl_en_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_main_meas_ctrl_shadowed_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_usb_meas_ctrl_en_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2262178086 651782409"
+INSTANCE: tb.dut.u_reg.u_usb_meas_ctrl_shadowed_cdc
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_aes_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_hmac_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_kmac_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_otbn_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "489161985 2298821864"
+INSTANCE: tb.dut.u_io_meas.u_meas
+ANNOTATION: "VC_COV_UNR"
+Block 17 "2798412932" ";"
+CHECKSUM: "489161985 2298821864"
+INSTANCE: tb.dut.u_io_div2_meas.u_meas
+ANNOTATION: "VC_COV_UNR"
+Block 17 "2798412932" ";"
+CHECKSUM: "489161985 2298821864"
+INSTANCE: tb.dut.u_io_div4_meas.u_meas
+ANNOTATION: "VC_COV_UNR"
+Block 17 "2798412932" ";"
+CHECKSUM: "489161985 2298821864"
+INSTANCE: tb.dut.u_main_meas.u_meas
+ANNOTATION: "VC_COV_UNR"
+Block 17 "2798412932" ";"
+CHECKSUM: "489161985 2298821864"
+INSTANCE: tb.dut.u_usb_meas.u_meas
+ANNOTATION: "VC_COV_UNR"
+Block 17 "2798412932" ";"
+CHECKSUM: "1907760401 2723074285"
+INSTANCE: tb.dut.u_reg.u_io_meas_ctrl_shadowed_lo_err_update_sync
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "3708980842" "(((!dst_pulse_o)) && src_active_flag) 1 -1" (1 "01")
```

Comment:
huh..this looks like fake logic i added for assertions. Should i be doing anything to help UNR understand that?

Reply:
This is the UNR genereated exclusion file, so it was able to sort it out.

Reply:
sorry what i meant was, i should have added this logic under `INC_ASSERT`, so I was wondering if UNR should even have looked at them? I guess it should since the coverage tool also looks at it...